src/reducers/index.js

```diff
@@ -75,7 +75,7 @@ import {
   VALIDATE_NETWORK_ATTRIBUTE_PENDING
 } from '../actions'
 
-export default combineReducers({
+export default combineReducers(injectSetState({
```

Comment:
the output of combineReducers here is also just one giant reducer, so rather than modifying each sub-reducer, you can do:
```js
const combinedReducers = combineReducers(...)
const handleSetState = (state = {}, { type, payload }) => type === SET_STATE ? payload : state
export default composeReducers(combinedReducers, handleSetState)
```